setup.py

```diff
@@ -8,6 +8,7 @@
     "numpy>=1.19.4",
     "mktestdocs>=0.1.0",
     "tqdm>=4.54.0",
+    "ray>=1.3.0"
```

Comment:
Let's add a new list here for the `ray` dependencies. Just like for `web_packages`.

Reply:
Done See c2a091386472d69dc9ce8b54e6b1c1434af70974